CHANGELOG.md

```diff
@@ -6,4 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.1]
```

Comment:
https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase actually recommends starting at 0.1.0, although https://keepachangelog.com has an example changelog that starts at 0.0.1, and we started at 0.0.1 for <https://github.com/ASFHyP3/sentinel1-orbits>. But I think I slightly prefer 0.1.0.
```suggestion
## [0.1.0]
```